engine/flink/test-util/src/main/scala/pl/touk/nussknacker/engine/flink/test/MiniClusterExecutionEnvironment.scala

```diff
@@ -18,10 +18,6 @@ import scala.collection.JavaConverters._
 class MiniClusterExecutionEnvironment(flinkMiniClusterHolder: FlinkMiniClusterHolder, userFlinkClusterConfig: Configuration, envConfig: AdditionalEnvironmentConfig) extends StreamExecutionEnvironment
   with LazyLogging with Matchers {
 
-  def runningJobs(): Iterable[JobID] = {
```

Comment:
Why you've removed it? It was a helpfull method...

Reply:
I will add it in FlinkMiniClusterHolder

Reply:
You should delegate to this method instead of removing it from here

Reply:
I wil add for back compability with `@Depricated`.

Reply:
Why you've added deprecated? What is wrong with this method?